### PR TITLE
added plots and fixed minor issues in astropy.units docs

### PR DESCRIPTION
**Pull request by mirca**
Does that look okay? #4487
capture d ecran 2016-01-25 a 10 24 10
capture d ecran 2016-01-25 a 10 24 21

### CODE REVIEW

**Comment by astrofrog** [comment]
@mirca - thanks! I think you can also include ``:include-source:`` in the ``plot`` directives to avoid having to repeat the example twice (the source from the actual plot example will be shown)

File: docs/units/quantity.rst [hunk 1]
```diff
@@ -100,21 +100,40 @@ the quantity:
...
+    from astropy import units as u
+    from astropy.visualization import quantity_support
+    quantity_support()
+    from matplotlib import pyplot as plt
+    plt.figure(figsize=(5,3))
+    plt.plot([1, 2, 3] * u.m)
+
 Quantities are automatically converted to the first unit set on a
 particular axis, so in the following, the y-axis remains in ``m`` even
-though the second line is given in ``cm``::
```
Comment (mhvk) [inline docs/units/quantity.rst]: I think the double colon `::` is used to indicate that the next block is code. I think it is not necessary given the `>>>`, but do check that the output documentation looks good.

**Comment by mhvk** [comment]
Looks nice!

**Comment by mirca** [comment]
@mhvk Thanks!

**Comment by mwcraig** [comment]
Restarted broken doc build

**Comment by mirca** [comment]
@astrofrog @mhvk any further thoughts on this?

**Comment by astrofrog** [comment]
@mirca - looks good, thanks! Can you add a changelog entry to the 1.1.2 section?

**Comment by mirca** [comment]
@astrofrog :+1: (:

**Pull request merged**